packages/astro/src/vite-plugin-mdx/index.ts

```diff
@@ -102,7 +102,8 @@ export default function mdxVitePlugin({ settings }: AstroPluginJSXOptions): Plug
 			}
 			id = removeQueryString(id);
 			// Shortcut: only use Astro renderer for MD and MDX files
-			if (!id.endsWith('.mdx')) {
+			const markdownExtensions = ['.md', '.mdx'];
+			if (markdownExtensions.every((ext) => !id.endsWith(ext))) {
```

Comment:
not sure if written as `not any` would be easier to understand than current `all not`, i.e.

```ts
if (!markdownExtensions.some((ext) => id.endsWith(ext)))
```

the former more closely resembles `vite-plugin-markdown` (`if (!isMarkdownFile(...))`), the latter more closely resembles the original code in this file.